apm-sniffer/optional-plugins/pom.xml

```diff
@@ -98,11 +100,26 @@
                         <configuration>
                             <tasks>
                                 <taskdef resource="net/sf/antcontrib/antcontrib.properties" classpathref="maven.runtime.classpath" />
+                                <mkdir dir="${plugin.dest.dir}" />
```

Comment:
Please read the pom.xml in `apm-sdk-plugin` module and plugins in it. Create pom properties `agent.package.dest.dir` and `optional.agent.package.dest.dir` to do the copy, and let copy op could be override by the plugin pom.

Only copy jar file is required, the setting file and readme are optional.

Reply:
I get it